arlas-core/src/main/java/io/arlas/server/app/ArlasAuthConfiguration.java

```diff
@@ -54,10 +57,20 @@
 
     private String publicRegex;
     public String getPublicRegex() {
+        // [swagger.*:*, persist.*:GET/POST/DELETE}]
+        List<String> pathToVerbs = this.publicUris.stream().map(uri -> {
+            String path = uri.split(":")[0];
+            String verbs = uri.split(":")[1];
```

Comment:
Maybe we should check if the uri.split(':') length is >= 2 to avoid an eventuak `IndexOutofbounds` exception
unless we are sure that the uri will always have the 'verb' part

Reply:
A check is present in configuration validation